src/wizard/ionic/index.md

```diff
@@ -0,0 +1,217 @@
+---
+name: Capacitor
+doc_link: https://docs.sentry.io/platforms/javascript/guides/capacitor/
+support_level: production
+type: framework
+---
+
+Install the Sentry Capacitor SDK alongside the sibling Sentry SDK:
+
+```angular {tabTitle: Ionic + Angular}
+# npm
+npm install --save @sentry/capacitor @sentry/angular
+
+# yarn
+yarn add @sentry/capacitor @sentry/angular @sentry/tracing
+```
+
+```react {tabTitle: Ionic + React}
+# npm
+npm install --save @sentry/capacitor @sentry/react 
+
+# yarn
+yarn add @sentry/capacitor @sentry/react  @sentry/tracing
+```
+
+```vue {tabTitle: Ionic + Vue}
```

Comment:
This tab control doesn't work on markdown. Only on mdx (which we use for the docs).

Please just use a single option not to pollute too much. (yarn is probably OK)

Reply:
Didn't know Wizard wouldn't support that :( 
Replaced the 3 samples with a single one explaining what to do when using each of them.